jest.config.ts

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   verbose: true,
-  prettierPath: 'prettier'
+  prettierPath: 'prettier',
```

Comment:
Took out prettier.  This should really be in peoplePicker PR.  not sure how you want to handle it.